acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/CmdLineParamsDefaultImpl.java

```diff
@@ -97,6 +98,19 @@ public List<String> createCmdLineParams() {
         params.add("--azure-tenant-id");
         params.add(azureParams.getTenantId());
       }
+      if (signerConfig.getKeystoresParameters().isPresent()) {
```

Comment:
Don't forget to add same to `tech.pegasys.web3signer.dsl.signer.runner.CmdLineParamsConfigFileImpl` as well. AT CLI DSL uses both config and and standard cli options.